docs/api/system-preferences.md

```diff
@@ -90,13 +90,14 @@ that contains the user information dictionary sent along with the notification.
 * `callback` Function
   * `event` String
   * `userInfo` Object
+  * `object` String
 
 Returns `Number` - The ID of this subscription
 
 Subscribes to native notifications of macOS, `callback` will be called with
 `callback(event, userInfo)` when the corresponding `event` happens. The
 `userInfo` is an Object that contains the user information dictionary sent
-along with the notification.
+along with the notification. The `object` is the sender of the notification.
```

Comment:
It seems that the `object` is only meaningful when it is a `NSString`?

I'm good with only supporting strings for now, but the documentation should explicitly document that "only `NSSString` is supported for now".

Reply:
Ahh, yes, you're right. I hadn't spotted that other types might be used.

I've updated the docs to make this limitation clear.

Thanks for the helpful feedback!